### DISCUSSION TITLE
Blog Post [8]

### DISCUSSION THREAD

**Discussion by gordi76** [NicoReimer/Breakpoint#17] [Weekly-Blog]
Hier sind die aktuellen UML-Modelle unseres Wiki-Projekts:

- Klassendiagramm: Zeigt die Struktur unserer Hauptkomponenten (Wiki, Page, User, etc.).

- Sequenzdiagramm: Beschreibt den Ablauf beim Erstellen einer neuen Wiki-Seite.

- Komponentendiagramm: Visualisiert die Kommunikation zwischen Frontend, Backend und Datenbank.

- Einsatzdiagramm: Stellt die physische Verteilung der Komponenten dar.

Hier findet ihr das [SAD](https://github.com/NicoReimer/Breakpoint/blob/main/Info/Software%20Architecture%20Document.pdf).

**Comment by lucariebel** [comment]
Hi,

Euer SAD bietet bisher eine gute Übersicht über die UML-Modelle und die Hauptkomponenten des Wiki-Projekts. Besonders gelungen sind die ausführlichen Sequenzdiagramme und die Beschreibung dazu, so wie auch das Klassendiagramm.

Viele Grüße 
euer WebShop24
